app/containers/Encrypt/Encrypt.jsx

```diff
@@ -0,0 +1,26 @@
+// @flow
+import React from 'react'
+
+import EncryptPanel from '../../components/Settings/EncryptPanel'
+import { ROUTES } from '../../core/constants'
+import styles from './Encrypt.scss'
+
+type Props = {
+  history: Object
+}
+
+export default class Encrypt extends React.Component<Props> {
+  componentWillMount = () => {
+    this.props.history.push(ROUTES.ENCRYPT)
```

Comment:
I dont think we want this here... shouldnt we just use a NavLink component to get here when the user clicks the encrypt key link https://github.com/CityOfZion/neon-wallet/blob/design-v2/design/_Dark/9.0-Settings-Dark.jpg

Reply:
Yeah, your right its handled in the NavLink component. Should have noticed as it throws a console warning about pushing the same path onto the history stack